cts/projection/positional.json

```diff
@@ -0,0 +1,88 @@
+{
+  "number": {
+    "request": {
+      "find": "composites",
+      "filter": {
+        "v": {
+          "$elemMatch": {
```

Comment:
Any reason we need to use `$elemMatch` filter before applying positional operator?

Reply:
Why do you need to limit the response size? Isn't it valuable to test for each one in composite collection?

Reply:
Sorry misunderstood the question. We need `$elemMatch` as positional operator requires it to be included. I've changed the filter to more general one, so it matches all of the documents.

Reply:
No need to be sorry :hugs:

Let me re-phrase the question. We are testing `projection` positional operator here correct? Why do we use filter at all? I think there is a reason, because otherwise it doesn't need be. Possibly something like you would get error or something?

Reply:
The empty filter would return error:
```
positional operator '.$' couldn't find a matching element in the array"
```

We need some condition that matches the element.

Reply:
I understand it now, thanks a lot :pray: I see you are using `$elemMatch` to match arrays, and the condition is not relevant, any array is okay right?

Could you use `$type` operator to match array type `{ $type: "array" }` in your filter instead? This way, it explicitly specifies array type, plus it applies to all arrays instead of ones greater than 42. What do you think? :thinking:

Reply:
The positional operator matches the filter against array **element** instead of the array itself. Thus, setting it to `{$type: "array"}`, would match only nested arrays. The current code checks if element exists, so it will return first element of each array in composites collection.